src/app/multiselect/multiselect.directive.js

```diff
@@ -186,6 +204,17 @@
                 }, true);
             }
 
+
+            /**
+             * @ngdoc method
+             * @name amoMultiselect#isGroupEmpty
+             * @description Determines whether or not items are selected
+             * @returns {Boolean}
+             */
+            function isGroupEmpty(group) {
+                return $filter('amoMultiselect')($filter('filter')(self.options, self.search), 'grouping', group).length === 0;
```

Comment:
Minor, but you can inject `filterFilter` and `amoMultiselectFilter` – which reference the resulting filter functions directly.